controls/radmaskedinput/features/placeholder.md

```diff
@@ -20,7 +20,7 @@ The __Placeholder__ character is a character used to represent the absence of us
 #### __Figure 1 : Default placeholder__
 ![](images/RadMaskedTextBox_Features_Placeholder_Default.png)
 
-If you want to change the default __Placeholder__, you should set a __RadMaskedInput__ control's __Placeholder__ property, like in the example below:
+When the __Value__ of a __RadMaskedInput__ control contains underscore character, it will be removed when we use the default placeholder. This is because of the default placeholder of the control is underscore character __"_"__. This behavior can be workaround by changing the default __Placeholder__. In order to do that you can set a __RadMaskedInput__ control's __Placeholder__ property, like in __Example 1__.

```

Comment:
I think this is not entirely true when you use the new ValueMode - in some cases the placeholder is preserved in the value. Can you test this and include a note that describes this for the RadMaskedTextInput?

Also "can be workaround" is not grammatically correct. Use "You can workaround this behavior.." or "This behavior can be worked around..". By the way you can use "alter" instead "workaround". Workaround is a negative word mostly used for bugs. You can try with "You can alter this behavior..."